crates/rolldown/tests/rolldown/function/file/css/artifacts.snap

```diff
@@ -0,0 +1,19 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Assets
+
+## out.css
```

Comment:
This is not expected, we should have a error like this https://hyrious.me/esbuild-repl/?version=0.23.0&b=e%00entry.js%00import+%7B+parse+%7D+from+%27ext%27%0Aimport+%27.%2Ffoo%27%0Aimport+foo+from+%27.%2Ffile.txt%27%0Aconsole.log%28foo%29%0Aconsole.log%28parse%29&b=%00foo.css%00body+%7B%7D&b=%00file.txt%00console.log&b=%00file.js%00&o=--bundle+--tree-shaking%3Dfalse+--format%3Desm+--external%3Aext+--outfile%3Dout.css+--loader%3A.txt%3Dfile.

But, let's solve it in another PR.